src/v/kafka/server/handlers/metadata.cc

```diff
@@ -137,8 +153,10 @@ metadata_response::topic make_topic_response_from_topic_metadata(
     return tp;
 }
 
-static ss::future<metadata_response::topic>
-create_topic(request_context& ctx, model::topic&& topic) {
+static ss::future<metadata_response::topic> create_topic(
+  request_context& ctx,
+  model::topic&& topic,
+  const is_node_isolated_or_decomissioned is_node_isolated) {
```

Comment:
This is kind of strange: if we are isolated, why proceed to attempt to autocreate topics?